docs/src/components/ComponentPage/AndroidComponentPage.js

```diff
@@ -29,10 +29,11 @@ import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
 import ComponentPage from './ComponentPage';
 
 type Props = {
-  documentationId: string,
+  documentationId: ?string,
   readme: string,
   screenshots: [
     {
+      blurb: ?string,
```

Comment:
I missed this last time around, so any screenshots with accompanying blurbs weren't displayed. Fixed now (also made the same change to iOS and RN).